sql/select-statements.mdx

```diff
@@ -73,14 +73,43 @@ WHERE condition
 ### GROUP BY clause
 The `GROUP BY` clause is used to compute aggregates over some set of fields.
 All GROUP BY queries will implicitly include a time window,
-and if the input doesn't already have a time window, 
+and if the input doesn't already have a time window,
 it should be specified as one of the grouping fields.
 
 For example,
 
 ```sql
-SELECT 
+SELECT
     count(*) AS bids,
     count(distinct auction_id) AS distinct_auctions,
     tumble(interval '1 minute') AS window
-    FROM BIDS GROUP BY 3
+FROM BIDS GROUP BY 3
+```
+
+### UNNEST operator
+
+The `UNNEST` operator allows you to unnest arrays into multiple rows. This can be used
+as a normal scalar function with the following restrictions:
+
+* It may only appear in the `SELECT` clause
+* Only one array may be unnested per query
```

Comment:
I think this should be "per select statement", as you could have consecutive unnests, I believe.